src/Console/Commands/InitLaravelApiKit.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+namespace Supaapps\LaravelApiKit\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Encryption\Encrypter;
+use Illuminate\Support\Str;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Facades\Http;
+
+use function Laravel\Prompts\confirm;
+
+class InitLaravelApiKit extends Command
+{
+    protected $signature = 'init:laravel-api-kit';
+
+    protected $description = 'Initialize laravel api kit';
+
+    private ?string $gitRepo;
+
+    private ?string $dbName = null;
+
+    private ?string $dbPassword;
+
+    public function handle()
+    {
+        $this->gitRepo = $this->ask('What is git repo url ?');
```

Comment:
I think we need repo name not url, since we use it for docker image?

Reply:
Yes for docker image. Is there a chance repo will be outside supaapps namespace?

the usage: https://github.com/supaapps/nao-referrals-api/blob/6ae819a4d26c1ae9d8df1595339cd0b169b2681e/Dockerfile#L2